### ISSUE TITLE
Sprint 3 – Scrum Meeting (16-12-2016) 

### ISSUE THREAD

**Issue by juzplayplay** [mewind/RSDSupermarketPOS#18]
Hi guys, how's the progression today?

**Comment by mewind** [comment]
Erm just to inform you guys, just now when I checked the code noticed that there is a build error and mo, after clarify with KarCheng and we noticed that we missed out a calling of CalculateTotal( ) method in  Form Activation. We accidentally uploaded the code with frmPayment, which is a form that we actually just testing to implement yesterday but have not completed. I have removed the part of code and called the CalculateTotal( ) method just now.

**Comment by lkc442012** [comment]
Thanks Sin Nee for updating the code and notice the error.

**Comment by mewind** [comment]
You're welcome KarCheng. you also contributed and helped me a lot during in my coding. We are a team!

**Comment by juzplayplay** [comment]
Noted. so are you guys facing any problems when developing this module?

**Comment by huylin25** [comment]
Let me briefly explain on it, actually the main problem of this function is just it includes too much of passing of data which we may actually messed up with others, so we actually need the inspectation of you guys to make it free of bugs...

**Comment by lkc442012** [comment]
Yes, HuyLin is right, I think we may need to check the code a lot of times to make sure the code do not have bugs or some errors in logic, so I think might need the help from you guys. Please look for me if you guys need any clarification.

**Comment by mewind** [comment]
Of course, that's what we have been doing all these times. No worries!

**Comment by juzplayplay** [comment]
I'll also try to help you guys up, please don't hesitate to find me if you guys need more help, so can I ask, whether everything is on the right track now?

**Comment by lkc442012** [comment]
currently should have no problem.

**Comment by mewind** [comment]
Okay, so today I'll be developing the Update Payment function which allow the user to update the quantity of items they desired to buy, or remove a particular item from cart.

**Comment by juzplayplay** [comment]
Alright, so at this moment did any idea comes to your mind?

**Comment by mewind** [comment]
I've discussed a bit with HuyLin and KarCheng today, I think I roughly know what to do to develop this function. Should be okay.

**Comment by huylin25** [comment]
We will also help her in when developing this function.

**Comment by lkc442012** [comment]
Ya, since actually I've done my tasks in this sprint, I'll also help SinNee in the development of this function.

**Comment by mewind** [comment]
Thanks guys! Much appreciated.

**Comment by juzplayplay** [comment]
Well, so I think we can stop our meeting here and proceed with our work now. Thanks guys for your effort.

**Comment by huylin25** [comment]
You're welcome.

**Comment by lkc442012** [comment]
Welcome.

**Comment by mewind** [comment]
You are welcome
